_templates/component/new/index.ejs.t

```diff
@@ -28,10 +28,10 @@ import { connect } from "react-redux"
 import { MapStateToProps } from "../../types/MapStateToProps"
 <% } -%>
 
-type Props = Readonly<{ children?: React.ReactNode }>
+type Props = Readonly<{ name?: string }>
```

Comment:
We'll need to update `Props` on line `12` as well 👍 (I think this file could be cleaned up a bit so that `Props` wasn't defined redundantly, but that's outside the scope of what you're doing)